src/components/AwayStatusIndicator.vue

```diff
@@ -0,0 +1,76 @@
+<template>
+    <span
+        v-if="checkCap()"
+        :user="user"
+        :class="{ 'kiwi-awaystatusindicator-away': isUserAway(),
+                  'kiwi-awaystatusindicator-not-away': !isUserAway(),
+                  'kiwi-awaystatusindicator-self': isUserSelf() }"
+        class="kiwi-awaystatusindicator"
+        @click="toggleSelfAway()"
+    />
+</template>
+
+<script>
+'kiwi public';
+
+export default {
+    props: ['user'],
+    computed: {
+        doesNetworkHaveAwayNotifyCap() {
+            return this.$state.getActiveNetwork().ircClient.network.cap.isEnabled('away-notify');
+        },
+    },
+    methods: {
+        checkCap() {
+            return this.doesNetworkHaveAwayNotifyCap;
```

Comment:
Why is this simply wrapping the computed property? It could just use the computed property directly...

Reply:
Removed the unneeded function, moved the computed property into the methods and we now use that instead.